framework/src/araui/animation/ui_animation.c

```diff
@@ -38,6 +38,7 @@ static bool _ui_anim_rotate_func(ui_widget_t widget, ui_anim_t anim, uint32_t *d
 static bool _ui_anim_scale_func(ui_widget_t widget, ui_anim_t anim, uint32_t *dt);
 static bool _ui_anim_sequence_func(ui_widget_t widget, ui_anim_t anim, uint32_t *dt);
 static bool _ui_anim_spawn_func(ui_widget_t widget, ui_anim_t anim, uint32_t *dt);
+static int _qsort_compare(const void *a, const void *b);
```

Comment:
_qsort_compare is too general.
--> _comp_anim_duration

Reply:
fixed